test/uk/gov/hmrc/transitmovementsvalidator/controllers/MessagesControllerSpec.scala

```diff
@@ -165,30 +171,20 @@ class MessagesControllerSpec
         .thenAnswer(
           _ => EitherT.rightT[Future, ValidationError](())
         )
-      val sut     = new MessagesController(stubControllerComponents(), mockXmlValidationService, mockJsonValidationService)
+      val sut     = new MessagesController(stubControllerComponents(), mockXmlValidationService, mockJsonValidationService, mockObjectStoreService)
       val source  = Source.single(ByteString(validJson, StandardCharsets.UTF_8))
       val request = FakeRequest("POST", s"/messages/$validCode/validate/", FakeHeaders(Seq(CONTENT_TYPE -> MimeTypes.JSON)), source)
       val result  = sut.validate(validCode)(request)
 
       status(result) mustBe NO_CONTENT
     }
 
-    "on a valid JSON file, with no content type, must return Unsupported Media Type" in {
```

Comment:
why was this removed?

Reply:
This was combined with the x-object-store-uri test.